USAGE.md

```diff
@@ -1,13 +1,9 @@
 # How to use sequenceParser
 
 ## Definitions
-#### Sequence  
-It's many files with a common pattern of numbers.
-#### File  
-It's all file without numbers in the filname and sequences with only one file.
-#### Folders  
-Just folders.
-
+* Sequence: many files with a common pattern of numbers.
+* File: all file without numbers in the filename and sequences with only one file.
```

Comment:
The description was already bad, but maybe we can clean it a bit.
We should not use "sequence" to define a "file".
"sequences with only one file" is a really bad wording because by default in sequenceparser a sequence contains at least 2 files.

Proposition:
```
* Sequence: a collection of files with a common prefix and suffix around a varying number.
* File: a file alone which is not included in a file sequence
* Folder: All folders. Folders are never considered as a sequence.
```